db/migrate/20170816150506_change_division_id_in_loan_question.rb

```diff
@@ -0,0 +1,18 @@
+class ChangeDivisionIdInLoanQuestion < ActiveRecord::Migration
+  def up
+    # Change column from string to integer
+    LoanQuestion.all.each { |q| q.update_attribute(:division_id, nil) }
+    change_column :loan_questions, :division_id, :integer, using: 'division_id::integer'
+    LoanQuestion.all.each { |q| q.update_attribute(:division_id, 99) }
+
+    change_column_null :loan_questions, :division_id, false
+  end
+
+  def down
+    # Change column from integer to string
+    change_column :loan_questions, :division_id, :string
+    LoanQuestion.all.each { |q| q.update_attribute(:division_id, 99) }
+
+    change_column_null :loan_questions, :division_id, true
+  end
```

Comment:
I'm a bit confused as to why this column was created as a string first. Was it a mistake? If so, that's fine, but it would be better to simply fix the original migration instead of adding a new one.

Or if there is a reason please let me know.